File: Python/Q61.py
# Q61.py - Program to Add Two Distances (inch-feet) Using Structures
class Distance:
    def __init__(self, feet, inch):
        self.feet = feet
        self.inch = inch

def add_dist(d1, d2):
    total_inch = d1.inch + d2.inch
    extra_feet, inch = divmod(total_inch, 12)
    feet = d1.feet + d2.feet + extra_feet
    return Distance(feet, inch)

d1 = Distance(int(input("Feet: ")), float(input("Inch: ")))
d2 = Distance(int(input("Feet: ")), float(input("Inch: ")))
sum_dist = add_dist(d1, d2)
print(f"Sum: {sum_dist.feet} feet {sum_dist.inch} inch")

File: C++/Q61.cpp
// Q61.cpp
#include <iostream>
using namespace std;

struct Distance {
    int feet;
    float inch;
};

int main() {
    Distance d1, d2, sum;
    cout << "Enter 1st distance (feet and inch): ";
    cin >> d1.feet >> d1.inch;
    cout << "Enter 2nd distance (feet and inch): ";
    cin >> d2.feet >> d2.inch;

    sum.feet = d1.feet + d2.feet;
    sum.inch = d1.inch + d2.inch;
    if (sum.inch >= 12) {
        sum.inch -= 12;
        sum.feet++;
    }
    cout << "Sum of distances = " << sum.feet << " feet " << sum.inch << " inches";
    return 0;
}
